docs/exiting-developer-mode.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Exiting Developer Mode
+layout: default
+nav_order: 9
+---
+
+## Exiting Developer Mode
+
+
+{: .warning }
+Entering and exiting Developer mode will **wipe all** user data!
+Backup anything if you have to.
+
+### Disable GGB Flag
+"(placeholder text) insert guide to reset gbb flags to allow exiting developer mode"
```

Comment:
Should be as simple as:
1. booting into dev mode
2. accessing vt2 (or whatever the shell with sudo access is called)
3. run fw script
4. select the reset gbb flags option
5. reboot

pretty sure it was like that when i did it to my leona.